content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -261,7 +261,7 @@ is automatically injected into the system when you install Istio with mutual TLS
 Re-run the testing command above to confirm that it returns 200 after the rule is added:
 
 {{< text bash >}}
-$ TOKEN=$(kubectl describe secret $(kubectl get secrets | grep default-token | cut -f1 -d ' ' | head -1) | grep -E '^token' | cut -f2 -d':' | tr -d '\t')
+$ TOKEN=$(kubectl describe secret $(kubectl get secrets | grep default-token | cut -f1 -d ' ' | head -1) | grep -E '^token' | cut -f2 -d':' | tr -ds ' ' '\t')
```

Comment:
Doesn't `tr -ds ' ' '\t'` mean delete all `'  '` and truncate multiple `'\t'` into a single `\t`? Is that exactly what you want? The original command deleted `'\t'` chars.

Reply:
You could change to tr -d '\t' | tr -d ' ' keeping the original remove all tabs and then remove all spaces (including potentially those in the token). If you just want to remove the leading spaces, maybe tr -d '\t' | sed 's/^ *//g'

Reply:
Thank you @frankbu @ericvn!  I misunderstood usage of -s, thought it is multiple strings :)

I removed \t as it is not needed and retested the cmd.

PTAL

Reply:
Are you sure `\t` is not needed? Maybe on some other platforms it will have a `\t`. Just to be safe you can remove both with `-d ' \t'`

Reply:
Agree with @frankbu's last comment if instead you want to remove all tabs (which was being done before and probably shouldn't be removed) and all spaces (I don't think the token will have an embedded space).

Reply:
Ok, I've added it back.  Yes, it was not needed for me but perhaps needed for some other k8s versions.